### PR TITLE
Adding doc for building the project.

### PR DESCRIPTION
**Pull request by r12f**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,3 +33,18 @@ In order to provide more virtual and emulated resources with limited hardware re
 - Kubernetes cluster in virtual machines
 
 For ore detail design and information, please refer to the docs folder in this repository.
```
Comment (phudtran) [inline README.md:35]: Can you please change the line from ore to more?

**Review by phudtran — APPROVED** [review]
Hi Riff thanks for the contribution!
Just one small comment, otherwise LGTM!

File: README.md (hunk 1 above)
Comment (r12f) [reply README.md:35]: sounds great! i thought it was an acronym and was trying to find this module for quite a while XD

Comment (phudtran) [reply README.md:35]: 👍 thanks!

**Review by phudtran — APPROVED** [review]
LGTM

**Comment by r12f** [comment]
Woot! Thanks! and btw, it looks like I am not authorized to merge the pull request yet, so might need some help to click the button. : D

**Comment by r12f** [comment]
awesome! thanks!

**Pull request merged**